addons/stock/models/stock_quant.py

```diff
@@ -176,6 +186,32 @@ def _update_dict(date_by_quant, key, value):
         for quant in self:
             quant.last_count_date = date_by_quant.get((quant.location_id.id, quant.package_id.id, quant.product_id.id, quant.lot_id.id, quant.owner_id.id))
 
+    def _search_last_count_date(self, operator, value):
+        op = PY_OPERATORS.get(operator)
+        if not op:
+            return NotImplemented
+        domain = Domain([('state', '=', 'done'), ('is_inventory', '=', True)])
+        if value is not False:
+            if operator == '=':
+                domain &= Domain([('date', '>=', value), ('date', '<=', value)])
+            else:
+                domain &= Domain('date', operator, value)
+        groups = self.env['stock.move.line']._read_group(
+            domain=domain,
+            aggregates=['id:recordset'],
+        )
+        move_lines = groups[0][0]
+        quant_domain = Domain.AND([
+            Domain('product_id', 'in', move_lines.product_id.ids),
+            Domain('lot_id', 'in', move_lines.lot_id.ids + [False]),
+            Domain('owner_id', 'in', move_lines.owner_id.ids + [False]),
+            Domain('location_id', 'in', move_lines.location_id.ids + move_lines.location_dest_id.ids),
+            Domain.OR([Domain('package_id', 'in', move_lines.package_id.ids + [False]), Domain('package_id', 'in', move_lines.result_package_id.ids + [False])]),
+        ])
```

Comment:
```suggestion
            Domain('package_id', 'in', move_lines.package_id.ids + move_lines.result_package_id.ids + [False]),
```
Just a small change, otherwise everything seems good to me.